jenkins/bootstrap.py

```diff
@@ -107,7 +111,7 @@ def Checkout(repo, branch, pull):
     os.chdir(repo)
     # TODO(fejta): cache git calls
     Subprocess([
-        git, 'fetch', '--tags', 'https://github.com/%s' % repo, ref,
+        git, 'fetch', '--tags', Repo(repo), ref,
```

Comment:
It seems like it would be easier to pass the remote URL directly here, but this is fine for now.